srcpkgs/wlroots/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'wlroots'
 pkgname=wlroots
-version=0.16.0
+version=0.16.1
 revision=1
 build_style=meson
 # Follow upstream packaging recommendations:
```

Comment:
wlroots moved to freedesktop gitlab, so this link should be changed as well https://gitlab.freedesktop.org/wlroots/wlroots/-/wikis/Packaging-recommendations